app/Version.hs

```diff
@@ -17,7 +17,7 @@ import qualified Development.GitRev as GitRev
 -- prerelease identifier here (if any). When releasing a proper version, simply
 -- set this to an empty string.
 prerelease :: String
-prerelease = "-rc1"
+prerelease = "-alpha1"
```

Comment:
I don't know what convention we typically follow, but I'm used to seeing `-alpha.1` and `-rc.1` (with the dot)

Reply:
:man_shrugging: If we include the `.`, then `Version.parseVersion` (assuming I'm reading it right) will [list the two values separately](https://github.com/hdgarrood/purescript-versions/blob/v6.0.0/src/Data/Version.purs#L146). But the general question is, "Does it matter?"

Reply:
Practically, I think it only matters if we have more than 9 alpha releases, in which case per (my reading of) the SemVer spec, `-alpha10` will sort before `-alpha1`, but `-alpha.10` would sort after `-alpha.1`.

Reply:
(Our history on this seems [less than fully consistent](https://github.com/purescript/purescript/tags?after=v0.14.0).)

Reply:
Ok, dot it is!

Reply:
:laughing: https://github.com/purescript/purescript/runs/5382701148?check_suite_focus=true#step:7:20

So... `alpha01`?

Reply:
Oops, guess you can't do a dot in .cabal. In that case I'm happy with `alpha-1`.